runtime/runtime.go

```diff
@@ -958,6 +958,19 @@ func (r *interpreterRuntime) newCreateAccountFunction(
 		return trampoline.Done{Result: account}
 	}
 }
+func storageUsedGetFunction(addressValue interpreter.AddressValue, runtimeInterface Interface) func() interpreter.UInt64Value {
+	address := addressValue.ToAddress()
+	return func() interpreter.UInt64Value {
+		return interpreter.UInt64Value(runtimeInterface.GetStorageUsed(address))
```

Comment:
This call to the host environment should be wrapped with `wrapPanic`

Reply:
like this?

Reply:
yep, [`10f72f7` (#439)](https://github.com/onflow/cadence/pull/439/commits/10f72f7cce3d636982256b1682928f1728bd8d25#diff-bb8ad382f59edfae9297e446895a161036f39115189e67da9d9f377a684a8f82R965-R967) looks good!